scripts/check-for-expired-polly-recordings.js

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env node
```

Comment:
Polly has an [`expiresIn` setting](https://netflix.github.io/pollyjs/#/configuration?id=expiresin), which causes the recordings Polly has captured to automatically expire after a set time. Since we set `recordIfMissing: false`, this will cause tests with expired recordings to fail tests, which may force developers to recapture recordings for tests that they've never touched. For this reason, I've opted to write this script to check for expired recordings instead of using `expiresIn`. That way we could run this on CI or locally or wherever and it wouldn't interfere with development.